[PPM-285] Remove ACTION_CONTROL_CONTROLLER_PING_REQUEST/RESPONSE

VladyslavTupikin opened the pull request:
Need to remove next messages and all related source code:

```
ACTION_CONTROL_CONTROLLER_PING_REQUEST
ACTION_CONTROL_CONTROLLER_PING_RESPONSE
```

https://jira.prplfoundation.org/browse/PPM-285

Commit:
common: tlvf: Remove ACTION_CONTROL_CONTROLLER_PING_REQUEST/RESPONSE

Need to remove next messages:
ACTION_CONTROL_CONTROLLER_PING_REQUEST
ACTION_CONTROL_CONTROLLER_PING_RESPONSE

https://jira.prplfoundation.org/browse/PPM-285

Signed-off-by: Vladyslav Tupikin <v.tupikin@inango-systems.com>

Commit:
agent: son_slave: Remove useless code

Need to remove all related code for messages:
ACTION_CONTROL_CONTROLLER_PING_REQUEST
ACTION_CONTROL_CONTROLLER_PING_RESPONSE

Signed-off-by: Vladyslav Tupikin <v.tupikin@inango-systems.com>

Commit:
controller: son_management: Remove unused code

Need to remove all related code for messages:
ACTION_CONTROL_CONTROLLER_PING_REQUEST
ACTION_CONTROL_CONTROLLER_PING_RESPONSE

Signed-off-by: Vladyslav Tupikin <v.tupikin@inango-systems.com>

Commit:
controller: son_master: Remove unused code

Need to remove all related code for messages:
ACTION_CONTROL_CONTROLLER_PING_REQUEST
ACTION_CONTROL_CONTROLLER_PING_RESPONSE

Signed-off-by: Vladyslav Tupikin <v.tupikin@inango-systems.com>

Commit:
controller: cli: Remove unused cli commands

Signed-off-by: Vladyslav Tupikin <v.tupikin@inango-systems.com>

simonbaren commented:
You need to reorder the commits here.   
Your first commit removes the classes from the YAML files and only later commits remove the usage. It will not compile separately.

VladyslavTupikin commented:
I have reversed the commits, now I am removing implementation after - messages.

morantr commented:
This PR should have 3 commits:
1. Remove all the code that use the messages you are going to delete.
2. Remove all messages from yaml files.
3. Update auto-generated code

mariomaz commented on controller/src/beerocks/master/son_master_thread.cpp:
```diff
@@ -2991,69 +2991,6 @@ bool master_thread::handle_cmdu_control_message(const std::string &src_mac,
-                    break;
-                }
-                memset(request->data(), 0, request->data_length());
-                if (!database.update_node_last_ping_sent(hostap_mac)) {
```
If `update_node_last_ping_sent` method is not used any more, then it should be removed too.
Please check the other database methods that were used in removed code

mariomaz reviewed (approved):
I think some database methods are not used anymore after the code removal

Commit:
controller: db: Remove deprecated update_node_last_ping_sent()

Need to remove deprecated method update_node_last_ping_sent()
from db source code.

Signed-off-by: Vladyslav Tupikin <v.tupikin@inango-systems.com>

arnout reviewed (approved):
I just *love* a pull request that removes 1000 lines of code :-) Progress!

Commit:
common: tlvf: Remove useless messages

Need to remove unused messages:
ACTION_CLI_PING_SLAVE_REQUEST
ACTION_CLI_PING_ALL_SLAVES_REQUEST

Signed-off-by: Vladyslav Tupikin <v.tupikin@inango-systems.com>
